tool/src/java/org/sakaiproject/gradebookng/tool/panels/AddGradeItemPanelContent.java

```diff
@@ -104,15 +112,39 @@ protected void onUpdate(AjaxRequestTarget target) {
 			}
         };
         released.setOutputMarkupId(true);
-        released.setEnabled(!assignment.getObject().isCounted());
+        released.setEnabled(!assignmentModel.getObject().isCounted());
         add(released);
         
         //if checked, release must also be checked and then disabled
-        final AjaxCheckBox counted = new AjaxCheckBox("counted", new PropertyModel<Boolean>(assignment, "counted")) {
+        final AjaxCheckBox counted = new AjaxCheckBox("counted", new PropertyModel<Boolean>(assignmentModel, "counted")) {
 			private static final long serialVersionUID = 1L;
 
 			@Override
 			protected void onUpdate(AjaxRequestTarget target) {
+				refreshState(target);
+			}
+
+			@Override
+			public void onEvent(IEvent<?> event) {
```

Comment:
Hm.. good question, I think I found the panel-scoped event a pretty good fit and ran with it! Plus I wasn't sure if making all the dynamic checkboxes "final"  was the best way to go.  On second thoughts, probably not too bad.

Happy to refactor.